pkg/assets/assets.go

```diff
@@ -0,0 +1,67 @@
+package assets
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/bootkube/pkg/assets/internal"
+)
+
+const (
+	AssetPathCAKey           = "ssl/ca.key"
```

Comment:
I'd be for us changing that everywhere then. The `.pem` suffix seems like a poor choice anyway. Of course its PEM format. One other difference I just noticed is `ssl` dir vs `tls`.

As long as we have the same names, the skew between systemd units / k8s manifests / Ignition/Cloud-config can be minimal.